debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/transforms/UnwrapFromMongoDbEnvelopeOperationsTestIT.java

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mongodb.transforms;
+
+import com.mongodb.MongoClient;
+import io.debezium.config.Configuration;
+import io.debezium.connector.mongodb.ConnectionContext.MongoPrimary;
+import io.debezium.connector.mongodb.*;
+import io.debezium.data.Envelope;
+import io.debezium.embedded.AbstractConnectorTest;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.bson.Document;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.util.Collections;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.BiConsumer;
+import java.util.function.Consumer;
+
+import static org.fest.assertions.Assertions.assertThat;
+import static org.junit.Assert.fail;
+
+/**
+ * Integration test for {@link UnwrapFromMongoDbEnvelope}. It sends operations into
+ * MongoDB and listens on messages that are generated by Debezium plug-in. The messages
+ * are then run through the SMT itself.
+ * <p>
+ * This tries to cover evey mongo update operation as described in the official documentation
+ * {@see https://docs.mongodb.com/v3.6/reference/operator/update/#id1}
+ *
+ * @author Renato Mefi
+ */
+public class UnwrapFromMongoDbEnvelopeOperationsTestIT extends AbstractConnectorTest {
+
+    private static final String DB_NAME = "transform_operations";
+    private static final String COLLECTION_NAME = "source";
+    private static final String TOPIC_NAME = "mongo.transform_operations.source";
+
+    private MongoDbTaskContext context;
+    private UnwrapFromMongoDbEnvelope<SourceRecord> transformation;
+
+    @Before
+    public void beforeEach() {
+        Debug.disable();
+        Print.disable();
+        stopConnector();
+        initializeConnectorTestFramework();
+
+        transformation = new UnwrapFromMongoDbEnvelope<>();
+        transformation.configure(Collections.emptyMap());
+
+        // Use the DB configuration to define the connector's configuration ...
+        Configuration config = TestHelper.getConfiguration().edit()
+                .with(MongoDbConnectorConfig.POLL_INTERVAL_MS, 10)
+                .with(MongoDbConnectorConfig.COLLECTION_WHITELIST, "transform_operations.*")
+                .with(MongoDbConnectorConfig.LOGICAL_NAME, "mongo")
+                .build();
+
+        // Set up the replication context for connections ...
+        context = new MongoDbTaskContext(config);
+
+        // Cleanup database
+        TestHelper.cleanDatabase(primary(), DB_NAME);
+
+        // Start the connector ...
+        start(MongoDbConnector.class, config);
+    }
+
+    @After
+    public void afterEach() {
+        try {
+            stopConnector();
+        } finally {
```

Comment:
Can you please move the `finally` to the next line? The project style is to have no code after closing braces (it's not like that everywhere yet, but we'd like to converge to that eventually).